README.md

```diff
@@ -29,14 +29,39 @@ make install
 ```
 
 Install location can be controlled by passing `DESTDIR=<DIR>` before the
-`install` target. You can also control the location of the installed environment
-files on the final target with `ENV_DIR`. The default is `/boot/efi/EFI/BOOT`
-which assumes that the [EFI
+`install` target. By default, the installation is integrated with the
+`/etc/grub.d` scripts, which means you need to run `grub-install` and
+`update-grub` after installation on the platform. This normally has to be done
+while running the device, but you can use
+[mender-convert](https://github.com/mendersoftware/mender-convert) as a way to
+do this at build time.
+
+If you want to install only the tools, or only the script and environment files,
```

Comment:
What are the "tools which one installs"? `grub-install` and those?

Reply:
Ah no, it's `grub-mender-grubenv-set` and `grub-mender-grubenv-print`. I should probably make that clear!